src/sentry/static/sentry/app/components/dateTime.jsx

```diff
@@ -62,7 +74,9 @@ class DateTime extends React.Component {
     }
 
     return (
-      <time {...carriedProps}>{moment.tz(date, options.timezone).format(format)}</time>
+      <time {...carriedProps}>
+        {this.state.momentFunc(date, options.timezone).format(format)}
```

Comment:
If I'm reading this correctly, this means we'll render the untranslated string first, and then when `moment-timezone` loads, each `DateTime`'s state will change triggering a re-render with the translations?